src/CsvTextEditor/Constants.cs

```diff
@@ -78,6 +78,10 @@ public static class Edit
 
             public const string DeleteLine = "Edit.DeleteLine";
             public static readonly InputGesture DeleteLineInputGesture = new InputGesture(Key.L, ModifierKeys.Control);
+            
+            public const string RemoveBlankLines = "Edit.RemoveBlankLines";
```

Comment:
also add the default input gestures (but set as null if not needed). this makes it easier for us to change this in the future.